subprojects/diagnostics/src/test/groovy/org/gradle/api/tasks/diagnostics/internal/dependencies/AsciiDependencyReportRendererTest.groovy

```diff
@@ -43,26 +43,22 @@ class AsciiDependencyReportRendererTest extends AbstractProjectBuilderSpec {
     }
 
     def "shows configuration header"() {
-        DeprecatableConfiguration configuration1 = Mock()
-        configuration1.getName() >> 'config1'
-        configuration1.getDescription() >> 'description'
-        configuration1.isCanBeResolved() >> true
-        DeprecatableConfiguration configuration2 = Mock()
-        configuration2.getName() >> 'config2'
-        configuration2.isCanBeResolved() >> true
+        given:
+        def conf1 = new ConfigurationDetails('config1', 'description', true, null, null)
+        def conf2 = new ConfigurationDetails('config2', null, false, null, null)
 
         when:
         renderer.prepareVisit()
-        renderer.startConfiguration(configuration1);
-        renderer.completeConfiguration(configuration1);
-        renderer.startConfiguration(configuration2);
-        renderer.completeConfiguration(configuration2);
+        renderer.startConfiguration(conf1);
+        renderer.completeConfiguration(conf1);
+        renderer.startConfiguration(conf2);
+        renderer.completeConfiguration(conf2);
 
         then:
         textOutput.value.readLines() == [
                 'config1 - description',
                 '',
-                'config2'
+                'config2 (n)'
```

Comment:
❓ `(n)`?

Reply:
The change in this test to use report models instead of mocked configurations triggered the change in output. `config2` is not resolvable and it is expected that its description includes `(n)`.